flake.nix

```diff
@@ -43,20 +43,9 @@
       targets.wasm32-unknown-unknown = false;
       targets.wasm32-wasip1 = false;
       targets.wasm32-wasip2 = false;
-
-      overrideVendorCargoPackage = {name, ...}: drv:
-        if name == "spiffe"
-        then
-          drv.overrideAttrs (_: {
-            patches = [
-              ./nix/patches/rust-spiffe.patch
```

Comment:
could you also remove the whole, now unused, `nix/patches` directory?

Reply:
That's already taken care of here: https://github.com/wasmCloud/wasmCloud/pull/4118/files#diff-bbc35ff8fbac31ea6b3dd9bcb7bddc1bb5bb597b171b937e3e5472a3496788b1